### PR TITLE
Build examples separately and provide node bindings

### PR DESCRIPTION
**Pull request by breathx**

### CODE REVIEW

File: scripts/src/run.sh [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+
+    node            runs gear-node
+    purge-chain     purges gear node chain
+    purge-dev-chain purges gear dev node chain
+
+EOF
+}
+
+run_node() {
+  RUST_LOG=gwasm=debug,gear=debug cargo run -p gear-node "$@"
```
Comment (grishasobol) [inline scripts/src/run.sh:24]:
```suggestion
  RUST_LOG="gwasm=debug,gear=debug" cargo run -p gear-node "$@"
```

Comment (breathx) [reply scripts/src/run.sh]: A forget change `gear` on `gear-core-backend`, yeah, but quotes aren't necessary here

Comment (grishasobol) [reply scripts/src/run.sh]: You can just add them and this string will be much more understandable

Comment (breathx) [reply scripts/src/run.sh]: Ok. As you wish

**Review by NikVolf — APPROVED** [review]

**Pull request merged**